src/isolate.rs

```diff
@@ -133,44 +140,25 @@ fn empty() -> libdeno::deno_buf {
 }
 
 impl Isolate {
-  pub fn new(
-    snapshot: libdeno::deno_buf,
-    flags: flags::DenoFlags,
-    argv_rest: Vec<String>,
-    dispatch: Dispatch,
-  ) -> Self {
+  pub fn new(state: Arc<IsolateState>, dispatch: Dispatch) -> Self {
     DENO_INIT.call_once(|| {
       unsafe { libdeno::deno_init() };
     });
     let shared = empty(); // TODO Use shared for message passing.
+    let snapshot = unsafe { super::snapshot::deno_snapshot.clone() };
```

Comment:
The snapshot should be passed as an argument to `Isolate::new()`

Reply:
Fixed. Thanks!